src/shared-components/dropdown/mobileDropdown.tsx

```diff
@@ -4,9 +4,21 @@ import PropTypes from 'prop-types';
 import ChevronIcon from '../../svgs/icons/chevron-icon.svg';
 import { DropdownContainer, dropdownInputStyle, IconContainer } from './style';
 
+import { OptionType } from './index';
+
+type MobileDropdownProps = {
+  value: string | null;
+  options: OptionType[];
+  textAlign: 'left' | 'center';
+  onSelectChange: (event: any) => void;
```

Comment:
```suggestion
  onSelectChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
```

I think this is only ever being used with `onSelectChange` as defined in the main Dropdown component, and it would be ideal, I think, to raise errors should we ever modify what's going on in the parent component.